packages/map-template/src/hooks/useDirectionsInfo.js

```diff
@@ -5,16 +5,18 @@ import directionsResponseState from '../atoms/directionsResponseState';
 import hasFoundRouteState from '../atoms/hasFoundRouteState';
 
 /*
- * Hook to handle when both origin location and destination location are selected, 
+ * Hook to handle when both origin location and destination location are selected,
  * and have geometry, call the MapsIndoors SDK to get information about the route.
  */
 const useDirectionsInfo = (originLocation, destinationLocation, directionsService, travelMode, accessibilityOn) => {
     const [totalDistance, setTotalDistance] = useState()
     const [totalTime, setTotalTime] = useState();
     const [hasFoundRoute, setHasFoundRoute] = useRecoilState(hasFoundRouteState);
     const [, setDirectionsResponse] = useRecoilState(directionsResponseState);
+    const [isDirectionReady, setIsDirectionReady] = useState()
```

Comment:
```suggestion
    const [areDirectionsReady, setAreDirectionsReady] = useState()
```
We usually use the plural form, "directions".

Reply:
Done ✅

Reply:
I still see `isDirectionReady` in the code.

Reply:
Please try now. I might have not pushed changes for some reason @ammapspeople